srcpkgs/bmake/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'bmake'
 pkgname=bmake
-version=20200402
+version=20200506
 revision=1
 create_wrksrc=yes
 short_desc="Portable version of the NetBSD make build tool"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="http://www.crufty.net/help/sjg/bmake.html"
 distfiles="http://www.crufty.net/ftp/pub/sjg/bmake-${version}.tar.gz"
-checksum=cf15f204ad8eea3396c2c7179b5ec0cffb06c9628b6f91050c6f2cbcaabb8928
+checksum=0ac33c9bb00f12e54a3571e492f8c88cd835f6df64533d008e188a47373538e2
 python_version=2 #unverified
```

Comment:
Looks like this can be set to `python_version=3`.

Reply:
Done, thanx!